Ensure that :checkhealth is happy with the Gentoo build type.
https://bugs.gentoo.org/757744
--- a/runtime/autoload/health/nvim.vim
+++ b/runtime/autoload/health/nvim.vim
@@ -118,7 +118,7 @@ function! s:check_performance() abort
   let buildtype = matchstr(execute('version'), '\v\cbuild type:?\s*[^\n\r\t ]+')
   if empty(buildtype)
     call health#report_error('failed to get build type from :version')
-  elseif buildtype =~# '\v(MinSizeRel|Release|RelWithDebInfo)'
+  elseif buildtype =~# '\v(MinSizeRel|Release|RelWithDebInfo|Gentoo)'
     call health#report_ok(buildtype)
   else
     call health#report_info(buildtype)
